packages/core/src/m3/functions.ts

```diff
@@ -48,6 +48,34 @@ const isContainment = (feature: Feature): feature is Containment =>
 const isReference = (feature: Feature): feature is Reference =>
     feature instanceof Reference
 
+const isMultiple = (feature: Feature): feature is Link =>
+    feature instanceof Link && feature.multiple
+
+
+/**
+ * The (names of the) metatypes of a feature.
+ */
+type FeatureMetaType =
```

Comment:
Not a TS expert here, but would something like this be type-safe? I mean, protecting from typos in these strings?

```
const FeatureMetaMap = {
    Containment,
    Property,
    Reference,
} as const;

type FeatureMetaType = keyof typeof FeatureMetaMap;
```

Reply:
That's actually (effectively) the same thing. Using the sum type already prevents typos: a typo would prevent compilation.